src/algebra/pointwise.lean

```diff
@@ -813,6 +813,8 @@ by simp [finset.mul_def]
 @[to_additive, mono] lemma mul_subset_mul  (hs : s₁ ⊆ s₂) (ht : t₁ ⊆ t₂) : s₁ * t₁ ⊆ s₂ * t₂ :=
```

Comment:
I guess `mono, to_additive` doesn't work? Can we teach `additive` to copy `mono`?

Reply:
That is, add it to:
https://github.com/leanprover-community/mathlib/blob/c72e709eaffaab6b377f28be9fc07ec7914bae00/src/algebra/group/to_additive.lean#L524-L526
and
https://github.com/leanprover-community/mathlib/blob/c72e709eaffaab6b377f28be9fc07ec7914bae00/src/algebra/group/to_additive.lean#L329-L334

Reply:
I guess there's a warning about that here that you added in #9790:
https://github.com/leanprover-community/mathlib/blob/c72e709eaffaab6b377f28be9fc07ec7914bae00/src/algebra/group/to_additive.lean#L343-L344

Reply:
Yeah its unfortunately non-trivial to make to_additive handle mono, because it stores some data in a reflected format, so some fairly significant additions would have to be made to handle that sort of data in a user attribute.
It would also be nice to have to_additive handle classes/structures, which I would consider even higher priority than mono, but that requires more functions to be exposed from lean to add structures to the environment :(.